contracts/6/examples/CentralisedFeeManagerWithLimits.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity ^0.6.8;
```

Comment:
In other contracts 0.6.9 version is used and without a caret. Keep the version same in all contracts for consistency.